protostar/commands/test/test_command.py

```diff
@@ -134,6 +150,14 @@ async def run(self, args) -> TestingSummary:
             seed=args.seed,
             slowest_tests_to_report_count=args.report_slowest_tests,
         )
+        failed_tests_paths = [
+            str(failed_test.file_path.absolute()) for failed_test in summary.failed
+        ]
+        protostar_cache.persist(
+            self._project_root_path,
+            "test_results",
+            {"failed_tests": failed_tests_paths},
+        )
```

Comment:
- [ ] add test checking if `TestCommand` actually saves test results

Reply:
using short name in [6577565](https://github.com/software-mansion/protostar/pull/895/commits/6577565e63905cf1afe03ef81fbd1b7994dba622)

Reply:
> add test checking if `TestCommand` actually saves failed test results

Added in [Add test command test](https://github.com/software-mansion/protostar/pull/895/commits/a457ac8d7ef0223572e4d1dbcb3768a6e714062a). Please let me know whether I implemented this correctly.

Reply:
Ok the tests are broken, I'm going to try to use `CLIApp`

Reply:
I took a slightly different approach and did this in in [add cache test](https://github.com/software-mansion/protostar/pull/895/commits/203b4c1c13b4df098ab2301f2a918fdf753a70c0).

Please, let me know if that's a good place for this.

Reply:
The integration test looks fine.